apps/meteor/app/api/server/v1/email-inbox.ts

```diff
@@ -11,10 +11,10 @@ API.v1.addRoute(
 	'email-inbox.list',
 	{ authRequired: true },
 	{
-		get() {
+		async get() {
 			const { offset, count } = this.getPaginationItems();
-			const { sort, query } = this.parseJsonQuery();
-			const emailInboxes = Promise.await(findEmailInboxes({ userId: this.userId, query, pagination: { offset, count, sort } }));
+			const { sort, query }: { sort: {}; query: {} } = this.parseJsonQuery();
```

Comment:
shouldn't this get the right types as Record (for both)

Reply:
Yes, this should. But, `sort` and `query` return as **any**

Reply:
Updated